lib/perl/Genome/File/Vcf/VepConsequenceParser.pm

```diff
@@ -5,7 +5,7 @@ package Genome::File::Vcf::VepConsequenceParser;
 
 use Data::Dumper;
 use Carp qw/confess/;
-use CGI qw/unescape/;
+use URI::Escape;
```

Comment:
Even though it automatically exports ```uri_unescape()```, it's still good practice to put it in the import list
```
use URI::Escape qw(uri_unescape);
```

Did you figure out whether this data is sourced from an actual URL and whether uri_unescape is the right thing to do here?